Add prep content for self coordinate (#691)
CodeYourFuture/curriculum: common-content/en/module/how-our-curriculum-works/rhythm-of-the-week/index.md

+++
title="🥁 Rhythm of the week"
headless="true"
time= 30
[objectives]
    1="Identify when we prep during a typical sprint week"
    2="Identify when we work on the backlog during a typical sprint week"
    3="Identify when the in-person session takes place on a typical sprint week"
+++

> Self-coordinating as a community poses several challenges. One of these challenges involves working out not just what to do but _when_ to do it. This becomes even more challenging with members of the community working on different schedules. To further complicate matters we use a flipped classroom model - much different from a conventional educational setting.

Reviewer: ```suggestion
Self-coordinating as a community poses challenges. One challenge is working out not just _what_ to do but _when_ to do it. This gets harder when we are working on different schedules. To make things even harder, we use a flipped classroom model: very different from a regular school.
```

To solve this problem, we use the **rhythm of the week**.

```mermaid
gantt
   title Rhythm of the week!
   dateFormat  YYYY-MM-DD
   tickInterval 1day
   weekday tuesday
   axisFormat %A

   section Trainees
   Prep     :2024-01-09, 2d
   In-person session   :2024-01-13, 1d
   Backlog :2024-01-11, 5d
   Demo :2024-01-12, 1d
```

> This rhythm enables trainees to plan their work during the week whilst still allowing flexibility in people's different schedules. In a typical sprint week, **Tuesday is the starting point** with in-person sessions on **Saturdays serving as the mid-point in the week**. Sundays and Mondays are used for trainees to reflect on their learning and finish off any backlog tasks.

Reviewer: ```suggestion
This rhythm supports trainees to plan their work during the week while still allowing flexibility in people's different schedules. In a typical sprint week, **Tuesday is the starting point** with in-person sessions on **Saturdays serving as the mid-point in the week**.  We use Sundays and Mondays to reflect on our learning and finish off any backlog tasks.
```
